src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-flow-modal/rgw-multisite-sync-flow-modal.component.html

```diff
@@ -1,129 +1,121 @@
-            </div>
-          </div>
-          <div class="form-group row">
-            <label class="cd-col-form-label required"
-                   for="destination_zone"
-                   i18n>Destination Zone</label>
-            <div class="cd-col-form-input">
-              <ng-container *ngTemplateOutlet="sourceAndDestZone;context: { name: 'destination_zone', zones: zones }"></ng-container>
-            </div>
-          </div>
+<cds-modal size="sm"
+           [open]="open">
+  <cds-modal-header (closeSelect)="closeModal()">
+    <h3 cdsModalHeaderHeading
+        i18n>{{ action | titlecase }} {{ groupType | upperFirst }} Flow</h3>
+  </cds-modal-header>
+  <form name="flowForm"
+        #frm="ngForm"
+        [formGroup]="currentFormGroupContext"
+        novalidate>
+    <div cdsModalContent>
+      <!-- Flow Name-->
+      <div class="form-item">
+        <cds-text-label for="flow_id"
+                        [invalid]="!currentFormGroupContext.controls.flow_id.valid && currentFormGroupContext.controls.flow_id.dirty"
+                        [invalidText]="flowIdError"
+                        cdRequiredField="Name">
+        <ng-container i18n>Name</ng-container>
+        <input cdsText
+               type="text"
+               placeholder="Flow Name..."
+               i18n-placeholder
+               id="flow_id"
+               formControlName="flow_id"
+               autofocus
+               [invalid]="!currentFormGroupContext.controls.flow_id.valid && currentFormGroupContext.controls.flow_id.dirty">
+        </cds-text-label>
+        <ng-template #flowIdError>
+          <span *ngIf="currentFormGroupContext.showError('flow_id', formDir, 'required')"
+                class="invalid-feedback">
+            <ng-container i18n> This field is required. </ng-container>
+          </span>
         </ng-template>
       </div>
-      <div class="modal-footer">
-        <cd-form-button-panel (submitActionEvent)="submit()"
-                              [form]="currentFormGroupContext"
-                              [submitText]="(action | titlecase) + ' ' + (groupType | upperFirst) + ' ' + 'Flow'"></cd-form-button-panel>
+      <!-- Bucket Name-->
+      <div class="form-item">
+        <cds-text-label for="bucket_name"
+                        [invalid]="!currentFormGroupContext.controls.bucket_name.valid && currentFormGroupContext.controls.bucket_name.dirty"
+                        [invalidText]="bucketNameError">
+          <ng-container i18n>Bucket Name</ng-container>
+          <input cdsText
+                 type="text"
+                 placeholder="Bucket Name..."
+                 i18n-placeholder
+                 id="bucket_name"
+                 formControlName="bucket_name"
+                 [invalid]="!currentFormGroupContext.controls.bucket_name.valid && currentFormGroupContext.controls.bucket_name.dirty">
```

Comment:
In case of no bucket name this shows a placeholder of `Bucket Name...` as you if were expected to input a value, but the input field is disabled, this is confusing.

Also, we should consider switching to readonly instead of disabling the field

https://carbondesignsystem.com/components/text-input/usage/#states

https://carbondesignsystem.com/patterns/read-only-states-pattern/#best-practices